5_databases/week3/database.sql

```diff
@@ -0,0 +1,64 @@
+DROP DATABASE week3_1;
+CREATE DATABASE week3_1 
+DEFAULT CHARACTER SET 'utf8mb4';
+
+USE week3_1;
+
+CREATE TABLE `meal` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` varchar(255) NOT NULL,
+  `description` text NULL DEFAULT NULL,
+  `location` varchar(255) NOT NULL,
+  `when` DATETIME NOT NULL,
+  `max_reservations` int(10) unsigned NOT NULL,
+  `price` decimal,
+  `created_date` DATETIME NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `reservation` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `number_of_guests` varchar(255) NOT NULL,
+  `meal_id` int(10) unsigned,
+  `created_date` DATETIME NOT NULL,
+  `contact_phonenumber` varchar(255) NULL,
+  `contact_name` varchar(255) NOT NULL,
+  `contact_email` varchar(255) NOT NULL,
+CONSTRAINT `fk_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE 
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `review` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+  `title` varchar(255) NOT NULL,
+  `description` text NULL DEFAULT NULL,
+  `meal_id` int(10) unsigned NOT NULL,
+  `stars` int(10) unsigned NOT NULL,
+  `created_date` DATETIME NOT NULL,
+  CONSTRAINT `fk_review_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
Good job! Looks correct. It is good that you considered what would make sense to be NULL and what should be required. 

If omitted, MySQL assumes a column to be NULL, however it is always a good idea to make it explicit for others to know. So remember to add it for all columns, i.e. `meal_id` or `price`